policyengine_us/parameters/gov/states/wv/tax/income/rates/head.yaml

```diff
@@ -0,0 +1,34 @@
+description: West Virginia taxes income at these rates for head of household filers.
+metadata:
+  type: marginal_rate
+  threshold_unit: currency-USD
+  rate_unit: /1
+  name: wv_income_tax_rate_head
```

Comment:
```suggestion
```

Reply:
(and elsewhere - we deprecated this attribute)